app/build.gradle.kts

```diff
@@ -275,6 +274,5 @@ dependencies {
     testImplementation(libs.test.turbine)
     testImplementation(projects.libraries.matrix.test)
 
-    ksp(libs.showkase.processor)
-    koverDependencies()
+        koverDependencies()
```

Comment:
Remove extra spaces?

Reply:
https://github.com/element-hq/element-x-android/pull/3125/commits/e641ea414ac43cb2bfe3fde255c67ab4e7b717a0 it seems like a batch replace broke this, and `ktlintformat` didn't help 🫤 .